### ISSUE TITLE
Scenario outline with certain syntax in table leads to tests being skipped in some behavex configurations

### ISSUE THREAD

**Issue by jbridger** [hrcorval/behavex#64]
**Describe the bug**
We found that tests will be skipped with certain `behavex` configuration parameters and where a scenario outline's `Examples:` contains additional text afterwards. E.g. `Examples: blah`. This led to some tests being skipped without us realizing for a while.

**To Reproduce**
Example scenario that will be skipped under certain conditions:
```
Feature: Test

  Scenario Outline: scenario 1
    Given <something>
    Examples: blah
      | something   |
      | something_a |
```

Scenario will be skipped if these conditions are true:
* Number of parallel processes is more than 1
* Parallel scheme is `scenario`

With `behavex`, these won't be skipped if:
* Use parallel scheme of `feature` instead of `scenario`
* Number of parallel processes is 1
* Removing the extra text after `Examples:`

**Expected behavior**
It's unclear whether this is invalid syntax, however these tests are run successfully with `behave`. 

As `behavex` is supposed to be a wrapper around `behave` then I think these tests should be allowed to run. Otherwise I would prefer the tests to fail rather than be skipped.

**Desktop (please complete the following information):**
 - OS: MacOS Ventura 13.1
 - Browser: Chrome
 - BehaveX version: 1.6.0
 - Python version: 3.10.8

**Comment by anibalinn** [comment]
Hi @jbridger, thanks for posting the issue. I have implemented the fix in release 2.0.0:
https://github.com/hrcorval/behavex/compare/release_2.0.0

I think it will be released by end of march. Thanks!!

**Comment by anibalinn** [comment]
Changes delivered in latest release (v2.0.1)
